<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirmation</class>
 <widget class="QDialog" name="Confirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirmation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radio_text">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Use License Text：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_license_help">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;  to use a validate license text to confirmation the software is genuine. or you can contact Qianfan company for help.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_license">
     <property name="minimumSize">
      <size>
       <width>540</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>720</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_file">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Use License File：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_file_help">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;  to use a validate license file to confirmation the software is genuine. or you can contact Qianfan company for help.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>File Path：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_path">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_path">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_buy">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Buy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Confirmation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
